app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(files_to_copy_path: str) -> None:
+    files = files_to_copy_path.split()
```

Comment:
Ensure that the input string contains exactly three elements separated by spaces, with `cp` as the first element. This is a requirement according to the task checklist.